src/main/java/dev/shivamnagpal/users/exceptions/handlers/GlobalExceptionHandler.java

```diff
@@ -21,7 +21,7 @@ public static void handle(Throwable throwable, HttpServerResponse httpServerResp
         log.error(throwable.getMessage(), throwable);
         if (throwable instanceof RestException restException) {
             ResponseWrapper<Object> failureResponseWrapper = ResponseWrapper
-                    .failure(restException.getErrorResponses(), restException.getPayload());
+                    .failure(restException.getErrorResponses());
```

Comment:
Why did you remove the restException.getPayload() from here?
It is needed for one case where we send the OtpToken if the user tries to login and his account isn't verified

Reply:
To solve the issue
Create another failure() method overload and it should accept List<ErrorResponses> and T data

Reply:
fixed